test/aensDelegation.js

```diff
@@ -34,16 +38,16 @@ describe('AensDelegation', () => {
             assert.fail('Could not initialize contract instance');
         }
         // fund account of nameOwner with 4 AE (some funds are required to cover the name fee)
-        await aeSdk.spend(4, nameOwnerKeypair.publicKey, {
-            onAccount: mainKeypair,
-            denomination: AmountFormatter.AE_AMOUNT_FORMATS.AE
+        await aeSdk.spend(4, nameOwnerAddress, {
+            onAccount: mainAccount,
+            denomination: AE_AMOUNT_FORMATS.AE
         });
 
         // fund newOwner with 1 AE as there is currently an issue (shouldn't be required for delegated actions)
         // see https://github.com/aeternity/aeternity/issues/3674
-        await aeSdk.spend(1, newOwnerKeypair.publicKey, {
-            onAccount: mainKeypair,
-            denomination: AmountFormatter.AE_AMOUNT_FORMATS.AE
+        await aeSdk.spend(1, nameOwnerAddress, {
```

Comment:
this was a hard debugging 😅
```suggestion
        await aeSdk.spend(1, newOwnerAddress, {
```

Reply:
sorry @davidyuk! seems like I fucked it up. but strange error anyway